### PR TITLE
Create project using android studio

### PR DESCRIPTION
**Pull request by sagesmith-wf**
#### Changes
- Added Android Studio files to .gitignore since it is auto generated
- Added the build folder to .gitinore since it is auto generated
- Generated a default project

#### Please Review
@Kuroshii @Nactyle @angustomlinson @JWhitney406

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
```
Comment (Kuroshii) [inline gradlew]: Does this need to be in the project? Shouldn't this be packaged in already by your IDE, so we don't need it up here?

File: gradlew.bat [hunk 2]
```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off
```
Comment (Kuroshii) [inline gradlew.bat]: Ditto

File: app/proguard-rules.pro [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Add project specific ProGuard rules here.
```
Comment (sagersmith8) [inline app/proguard-rules.pro]: I am assuming we might want to use [ProGuard](https://developer.android.com/studio/build/shrink-code.html#shrink-resources) but I haven't really looked into it yet. It allows you to make your list of resources shorter somehow.

File: gradlew.bat (hunk 2 above)
Comment (sagersmith8) [inline gradlew.bat]: Same as above

**Pull request merged**